vita3k/shader/src/usse_utilities.cpp

```diff
@@ -863,6 +863,11 @@ spv::Id shader::usse::utils::load(spv::Builder &b, const SpirvShaderParameters &
     int highest_dest_write_offset = -1; ///< Highest offset of the component to write.
 
     for (int i = 0; i < 4; i++) {
+
+        if ((int)op.swizzle[i] >= (int)SwizzleChannel::_0) {
```

Comment:
static_cast, please. While you are at it I hope you can clean up the line below :(

Reply:
@pent0 I had to use renderdoc and it's shader debugger for it ;)